stairway/src/main/java/bio/terra/stairway/FlightEnumeration.java

```diff
@@ -0,0 +1,32 @@
+package bio.terra.stairway;
+
+import java.util.List;
+
+/** Class that holds the return value of the enhanced getFlights Stairway entrypoint */
+public class FlightEnumeration {
+  private final int totalFlights;
+  private final String nextPageToken;
+  private final List<FlightState> flightStateList;
```

Comment:
It's a matter of taste but I find it redundant to include an object's type in its name. Plural names are more readable IMO, and don't need changing if the underling type changes.
```suggestion
  private final List<FlightState> flightStates;
```